content/applications/sales/point_of_sale/payment_methods/terminals/stripe.rst

```diff
@@ -6,7 +6,11 @@ Connecting a payment terminal allows you to offer a fluid payment flow to your c
 the work of your cashiers.
 
 .. seealso::
-   - :doc:`Use Stripe as payment provider. <../../../../finance/payment_providers/stripe>`
+   - :doc:`Using Stripe as payment provider documentation
+     <../../../../finance/payment_providers/stripe>`
+   - `List of countries supported on Stripe <https://stripe.com/global>`_
```

Comment:
Payment Terminals are not supported in every one of these countries: https://support.stripe.com/questions/global-availability-for-stripe-terminal